fastchat/utils.py

```diff
@@ -12,6 +13,7 @@
 import warnings
 
 import requests
+from PIL import Image
```

Comment:
Do not import this here because this introduces a new dependency.

Apply the above rule to all places. Users who do not want to use Llava should be able to run FastChat without any new dependency.